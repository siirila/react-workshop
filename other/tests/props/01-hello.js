import React from 'react'
import {render} from 'enzyme'
const SayHello = require(`../../../src/props/${process.env.TEST_DIR ||
  'exercises'}/01-hello`).default

test('Renders "Hello {props.firstName} {props.lastName}" in a div', () => {
  expect(render(<SayHello firstName="Jill" lastName="Doe" />)).toMatchSnapshot()
})
