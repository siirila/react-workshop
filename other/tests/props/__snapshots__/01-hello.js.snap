// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "Hello {props.firstName} {props.lastName}" in a div 1`] = `
<div>
  Hello Jill Doe
</div>
`;
